dev/com.ibm.websphere.appserver.features/visibility/public/mpRestClient-1.2/resources/l10n/com.ibm.websphere.appserver.mpRestClient-1.2_cs.properties

```diff
@@ -0,0 +1,18 @@
+#/*******************************************************************************
+# * Copyright (c) 2018 IBM Corporation and others.
+# * All rights reserved. This program and the accompanying materials
+# * are made available under the terms of the Eclipse Public License v1.0
+# * which accompanies this distribution, and is available at
+# * http://www.eclipse.org/legal/epl-v10.html
+# *
+# * Contributors:
+# *     IBM Corporation - initial API and implementation
+# *******************************************************************************/
+# -------------------------------------------------------------------------------------------------
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=Specifikace klienta MicroProfile Rest Client poskytuje typov\u011b bezpe\u010dn\u00fd mechanizmus pro vyvol\u00e1n\u00ed slu\u017eeb RESTful.
```

Comment:
Check with @pnickoll before checking in pre-translated files, even if they were copied from elsewhere.

Reply:
Phil replied:

> Copying files is fine, good to let me know though so when I send them off I know where they came from (I always have to check when I see new translated files)